mmtbx/refinement/ensemble_refinement/__init__.py

```diff
@@ -446,30 +463,40 @@ def __init__(self, fmodel,
         self.target_k1 = self.fmodel_running.scale_k1()
         self.update_normalisation_factors()
       else:
-        make_header("Calculate and fix scale of Ncalc", out = self.log)
-        self.fmodel_running.n_obs_n_calc(update_nobs_ncalc = True)
+        make_header("Calculate and fix scale of Ncalc", out=self.log)
+        self.fmodel_running.n_obs_n_calc(update_nobs_ncalc=True)
         print("Fix Ncalc scale          : True", file=self.log)
         print("Sum current Ncalc        : {0:5.3f}".format(sum(self.fmodel_running.n_calc)), file=self.log)
 
-    #Set ADP model
+    # Set ADP model
     self.tls_manager = er_tls_manager()
-    self.setup_tls_selections(tls_group_selection_strings = self.params.tls_group_selections)
-    self.fit_tls(input_model = self.model)
-    self.assign_solvent_tls_groups()
+    # Fit pTLS to starting atomic model
+    if self.params.import_tls_pdb is None:
+      self.setup_tls_selections(
+        tls_group_selection_strings=self.params.tls_group_selections)
+      self.fit_tls(input_model=self.model)
+      self.assign_solvent_tls_groups()
+    # Import TLS from reference model
+    else:
+      fit_tlsos, fit_tls_strings = self.import_tls_selections()
+      self.setup_tls_selections(tls_group_selection_strings=fit_tls_strings)
+      self.model.tls_groups.tlsos = fit_tlsos
+      self.tls_manager.tls_operators = fit_tlsos
+      self.assign_solvent_tls_groups()
```

Comment:
```suggestion
    else:
      # Import TLS from reference model
      fit_tlsos, fit_tls_strings = self.import_tls_selections()
      self.setup_tls_selections(tls_group_selection_strings=fit_tls_strings)
      self.model.tls_groups.tlsos = fit_tlsos
      self.tls_manager.tls_operators = fit_tlsos
    self.assign_solvent_tls_groups()
```
Just something I noticed scrolling past

Reply:
Thanks - fixed now